pkg/pillar/evetpm/tpm.go

```diff
@@ -76,6 +77,28 @@ const (
 	// TpmSavedDiskSealingPcrs is the file that holds a copy of PCR values
 	// at the time of generating and sealing the disk key into the TPM.
 	TpmSavedDiskSealingPcrs = types.PersistStatusDir + "/sealingpcrs"
+
+	// TpmEvtLogBinFile is a kernel exposed variable that contains the
+	// TPM event log.
+	TpmEvtLogBinFile = "binary_bios_measurements"
```

Comment:
I'm not a fan of inventing abbreviations like "evt" since it is hard to remember which letters different implementors chose to drop from names. 

AFAIU these names are local to this file thus we can drop the Tpm prefix and call it eventLogBinFile.